tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrerTest.xtend

```diff
@@ -0,0 +1,88 @@
+package org.testeditor.tcl.dsl.jvmmodel
+
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.dsl.common.testing.DummyFixture
+
+class TclJvmModelInferrerTest extends AbstractTclGeneratorIntegrationTest {
+
+	@Before
+	def void parseAmlModel() {
+		parseAml(DummyFixture.amlModel).assertNoSyntaxErrors
+	}
+
+	@Test
+	def void testGenerationForResolvedFixture() {
+		//given
+		val tclModel = parseTcl('''
```

Comment:
Add a test that uses macros / config to verify code generation in those cases, too?